docs/tools/code-extension.mdx

```diff
@@ -18,6 +18,22 @@ src/
 src-gen/Foo
 ```
 
+Overall, we encourage you to open your project in VS Code using the following structure to fully leverage the features of the VS Code extension:
+
+```
+├── root
```

Comment:
This stuff should be mentioned before the "creating a new file" stuff and should probably mention `lingo init`.

Reply:
I would rename the `Create new file` section to `Create new project` and include information on the correct project structure, `lingo init` command, along with details on the error encountered if this structure is missing.

Reply:
Yes, that makes sense, modulo the package/project discussion that we're having :-)

Reply:
```suggestion
├── .
```